src/utils/units.ts

```diff
@@ -101,6 +101,9 @@ const convertTokenBase = (value: TokenValue, oldDecimal: number, newDecimal: num
   return toTokenBase(fromTokenBase(value, oldDecimal), newDecimal);
 };
 
+export const getDecimals = (value: string) =>
+  value.includes('.') ? value.split('.')[1].length : 0;
```

Comment:
I don't know if we allow decimal values with `,` instead of `.`, but if we do, this won't always work

Reply:
We force `.`. Because my locale would normally use `,` 😄

Reply:
European powaaaaaaaaa